backend/backend.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"path/filepath"
+	"strconv"
+
+	"gopkg.in/ini.v1"
+
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/environ"
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/route"
+	"github.com/ucsdscheduleplanner/UCSD-Schedule-Planner/backend/store"
+)
+
+func readConfig(configFile string) (config *ini.File, err error) {
+	config, err = ini.Load(configFile)
+
+	if err != nil {
+		return nil, fmt.Errorf("Error reading config file: %v", err)
+	}
+
+	if config == nil {
+		return nil, fmt.Errorf("Could not find config file: %v", configFile)
+	}
+
+	return
+}
+
+func main() {
+
+	// config, err := readConfig(filepath.Join(".", "config", "config.example.ini"))
```

Comment:
Sorry for the confusion. Since I'm still using the data from previous quarters for testing, I need a config.dev.ini, which shouldn't be in the repo. Will squash and that that file out completely